src/main/java/mate/academy/booksstore/dto/CartItemRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.booksstore.dto;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class CartItemRequestDto {
+    @NotNull
```

Comment:
it's better to ident annotated fields:
```suggestion
    @NotNull
    private Long bookId;
    
    @Min(1)
    private Integer quantity;
```